docs/memlim.md

```diff
@@ -4,40 +4,32 @@ Each of the individual services within the sidecar has a default
 memory limit. The memory limit is a maximum number of bytes that a service is 
 allowed to consume. This is useful to prevent a single service from consuming
 all available memory on the container and causing other services to fail as a
-result. Currently, each "wire" service has a default memory limit of `512MB`
-while other services are limited to `128MB`.
+result.
 
-Users can override the default memory limits if desired by setting various 
-environment variables as detailed below.
+Refer to the [Memory Limits](https://cyral.com/docs/sidecars/deployment/memory-limits)
+page of our public docs for more details.
 
-## Environment Variables
+In order to change the memory limits for your sidecar container, override the
+default settings using the [environment variables](https://cyral.com/docs/sidecars/deployment/memory-limits#environment-variables)
+detailed in our public docs.
 
-The following environment variables can be set to override the default memory
-limits.
+## Setting memory limits via environment file
```

Comment:
Maybe put the environment variable session under the environment file title? That's just a nit though

Reply:
Just adjusted the whole text to make it clearer and avoid the idea that the example section for the environment file is the only way to do it.